spec/lib/vanagon/driver_spec.rb

```diff
@@ -54,6 +54,40 @@ def create_driver(platform, options = {})
                               'engine' => 'hardware' })
     end
 
+    it 'returns the platform and a scheduler engine when scheduled resources are enabled and an engine is specified' do
+      ENV['VANAGON_SCHEDULED_HARDWARE'] = "true"
+
+      platform = eval_platform('aix-7.1-ppc', <<-END)
+        platform 'aix-7.1-ppc' do |plat|
+          plat.build_host ["pe-aix-71-01", "pe-aix-71-02"]
+          plat.engine "scheduled"
+        end
+      END
+
+      info = create_driver(platform).build_host_info
+
+      expect(info).to match({ 'name'   => 'aix-7.1-ppc',
```

Comment:
Hm.  This is a good point. I was working from the platform names being used for hardware in vanagon, which are these dot-having names.

This *is* the place where we're expecting to emit abs-ready names, so I think it's useful to do the translation here while it's happening (as no other consumers exist for this).